tools/compilers/gcc-glibc/Makefile

```diff
@@ -8,8 +8,10 @@ GMP_VERSION := 5.1.3
 MPFR_VERSION := 3.1.2
 MPC_VERSION := 1.0.2
 
-# Defines where to install the toolchain
--include Makelocal
+# Define where to install the toolchain
```

Comment:
can you add a check to make sure AKAROS_TOOLCHAINS is not empty?  there are example checks in tools/Makefrag.

also, the commit needs an (XCC) tag.  I'd also put a line in the commit message that says "Delete your toolchain Makelocal and set AKAROS_TOOLCHAINS in your environment.  Rebuild your toolchain"  i'd also clarify the difference between AKAROS_TOOLCHAINS and the old AKAROS_XCC_ROOT.  XCC_ROOT was a particular toolchain (e.g. x86_64).  AKAROS_TOOLCHAINS is the directory for all toolchains, e.g. riscv, x86_64.  So TOOLCHAINS is the parent of the old XCC_ROOT.

Reply:
On it!

Reply:
still need a check for AKAROS_TOOLCHAINS being set or not.  can use something like the one in tools/Makefrag

Reply:
and i'd squash this fix into the "  Add AKAROS_TOOLCHAINS (XCC)" commit

Reply:
The latest commit fixes it.